servers/pollux/default.nix

```diff
@@ -0,0 +1,56 @@
+{ modulesPath, inputs, config, pkgs, ... }:
+{
+  imports = [
+    inputs.serokell-nix.nixosModules.ec2
+    inputs.hermetic.nixosModules.hermetic
+    inputs.self.nixosModules.suitecrm
+  ];
+
+  networking.firewall.allowedTCPPorts = [ 80 443 ];
+
+  nixpkgs.overlays = [ inputs.composition-c4.overlay ];
+
+  services.nginx.virtualHosts.suitecrm = {
+      serverName = "suitecrm.serokell.team";
+      default = true;
+
+      enableACME = true;
+      forceSSL = true;
+    };
+
+  services.suitecrm =
+    { enable = true;
+      suitecrmPackage = with pkgs; stdenv.mkDerivation rec {
```

Comment:
I don't really like how the package definition is inlined, but I guess it's fine for this use-case? Still would be a bit better if it was exported to the flake toplevel for debugging purposes

Reply:
tbh I agree, I'll move it now